src/Models/ClientOptions.cs

```diff
@@ -42,5 +43,11 @@ internal void OverrideWithEnvVars()
             if (!string.IsNullOrWhiteSpace(urlEnvVar))
                 BaseUrl = new Uri(urlEnvVar, UriKind.Absolute);
         }
+
+        internal void EnsureValid()
+        {
+            if (Timeout.TotalSeconds < 1 || Timeout.TotalSeconds > 60)
+                throw new InvalidClientOptionsException("Timeout must be between 1 and 60 seconds");
```

Comment:
60 is probably a bad value, let's ensure max is 59

Reply:
modified ✅